<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MazeView</class>
 <widget class="QWidget" name="MazeView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>678</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="vert_layout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Difficulty:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="b_difficulty">
         <item>
          <property name="text">
           <string>Easy</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Medium</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Hard</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="b_gen_maze">
         <property name="text">
          <string>Generate Maze</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="b_solve">
         <property name="text">
          <string>Solve</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="l_iter">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>N = 1</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>b_gen_maze</sender>
   <signal>clicked()</signal>
   <receiver>MazeView</receiver>
   <slot>gen_maze()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>314</x>
     <y>335</y>
    </hint>
    <hint type="destinationlabel">
     <x>382</x>
     <y>-18</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>b_solve</sender>
   <signal>clicked()</signal>
   <receiver>MazeView</receiver>
   <slot>solve()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>526</x>
     <y>341</y>
    </hint>
    <hint type="destinationlabel">
     <x>525</x>
     <y>-23</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>gen_maze()</slot>
  <slot>solve()</slot>
 </slots>
</ui>
